src/bin/pg_autoctl/pgsql.c

```diff
@@ -1623,36 +1646,36 @@ pgsql_replication_slot_maintain(PGSQL *pgsql, NodeAddressArray *nodeArray)
 	char values[BUFSIZE] = { 0 };
 
 	/* *INDENT-OFF* */
-	char *sqlTemplate =
-		"WITH nodes(slot_name, lsn) as ("
-		" SELECT '" REPLICATION_SLOT_NAME_DEFAULT "_' || id, lsn"
-		"   FROM (%s) as sb(id, lsn) "
-		"), \n"
-		"dropped as ("
-		" SELECT slot_name, pg_drop_replication_slot(slot_name) "
-		"   FROM pg_replication_slots pgrs LEFT JOIN nodes USING(slot_name) "
-		"  WHERE nodes.slot_name IS NULL "
-		"    AND slot_name ~ '" REPLICATION_SLOT_NAME_PATTERN "' "
-		"    AND not active"
-		"    AND slot_type = 'physical'"
-		"), \n"
-		"advanced as ("
-		"SELECT a.slot_name, a.end_lsn"
-		"  FROM pg_replication_slots s JOIN nodes USING(slot_name), "
-		"       LATERAL pg_replication_slot_advance(slot_name, lsn) a"
-		" WHERE nodes.lsn <> '0/0' and nodes.lsn >= s.restart_lsn "
-		"), \n"
-		"created as ("
-		"SELECT c.slot_name, c.lsn "
-		"  FROM nodes LEFT JOIN pg_replication_slots pgrs USING(slot_name), "
-		"       LATERAL pg_create_physical_replication_slot(slot_name, true) c"
-		" WHERE pgrs.slot_name IS NULL "
-		") \n"
-		"SELECT 'create', slot_name, lsn FROM created "
-		" union all "
-		"SELECT 'drop', slot_name, NULL::pg_lsn FROM dropped "
-		" union all "
-		"SELECT 'advance', slot_name, end_lsn FROM advanced ";
+    char *sqlTemplate =
+            "WITH nodes(slot_name, lsn) as ("
+            " SELECT '" REPLICATION_SLOT_NAME_DEFAULT "_' || id, lsn"
+            "   FROM (%s) as sb(id, lsn) "
+            "), \n"
+            "dropped as ("
+            " SELECT slot_name, pg_drop_replication_slot(slot_name) "
+            "   FROM pg_replication_slots pgrs LEFT JOIN nodes USING(slot_name) "
+            "  WHERE nodes.slot_name IS NULL "
+            "    AND slot_name ~ '" REPLICATION_SLOT_NAME_PATTERN "' "
+            "    AND not active"
+            "    AND slot_type = 'physical'"
+            "), \n"
+            "advanced as ("
+            "SELECT a.slot_name, a.end_lsn"
+            "  FROM pg_replication_slots s JOIN nodes USING(slot_name), "
+            "       LATERAL pg_replication_slot_advance(slot_name, lsn) a"
+            " WHERE nodes.lsn <> '0/0' and nodes.lsn >= s.restart_lsn "
+            "), \n"
+            "created as ("
+            "SELECT c.slot_name, c.lsn "
+            "  FROM nodes LEFT JOIN pg_replication_slots pgrs USING(slot_name), "
+            "       LATERAL pg_create_physical_replication_slot(slot_name, true) c"
+            " WHERE pgrs.slot_name IS NULL "
+            ") \n"
+            "SELECT 'create', slot_name, lsn FROM created "
+            " union all "
+            "SELECT 'drop', slot_name, NULL::pg_lsn FROM dropped "
+            " union all "
+            "SELECT 'advance', slot_name, end_lsn FROM advanced ";
```

Comment:
This looks like a whitespace change that we should not have in this PR, please fix.